lib/git-fastclone.rb

```diff
@@ -47,13 +47,23 @@ def reference_repo_dir(url, reference_dir, using_local_repo)
     end
     module_function :reference_repo_dir
 
+    def reference_filename(filename)
+      portation_character = if RbConfig::CONFIG['host_os'] =~ /mswin|msys|mingw|cygwin|bccwin|wince|emc/
+                              '_'
+                            else
+                              ':'
+                            end
+      "#{portation_character}#{filename}"
```

Comment:
Two minor nits - 
1. I think repos will not infrequently have underscores in their names, so I'd suggest using something that makes the separator a bit more clear.
2. `s/portation_character/separator/`

```suggestion
      separator = if RbConfig::CONFIG['host_os'] =~ /mswin|msys|mingw|cygwin|bccwin|wince|emc/
                              '__'
                            else
                              ':'
                            end
      "#{separator}#{filename}"
```

Reply:
I can agree on both of these changes, I think it would make things more clear